code/datums/jobs/squads.dm

```diff
@@ -386,30 +386,24 @@
 	var/text = copytext_char(sanitize(message), 1, MAX_MESSAGE_LEN)
 	if(ishuman(sender))
 		var/obj/item/card/id/ID = sender.get_idcard()
-		nametext = "[ID?.rank] [sender.name] transmits: "
+		nametext = "[ID?.rank] [sender.real_name] transmits: "
```

Comment:
Why?

Reply:
> Why?

Avoids clutter of '(Unknown) as Urist McOfficer'